pkg/scheduler/apis/config/scheme/scheme_test.go

```diff
@@ -66,6 +66,18 @@ profiles:
   - name: ServiceAffinity
     args:
       affinityLabels: ["bar"]
+  - name: NodeResourcesLeastAllocated
```

Comment:
Also test the default values, in the test named "empty and no plugin args" around line 250

Reply:
done